VotingApplication/VotingApplication.Data/Model/Event.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System;
+using System.ComponentModel.DataAnnotations;
+
+namespace VotingApplication.Data.Model
+{
+    public enum EventType
+    {
+        Error,
+        GoToPage,
+        CreatePoll,
+        UpdateResults,
+        SetExpiry,
+        SetPollType,
+        SetQuestion,
+        SetInviteOnly,
+        SetNamedVoting,
+        SetOptionAdding,
+        AddOption,
+        UpdateOption,
+        DeleteOption,
+        AddVote,
+        DeleteVote,
+        AddBallot,
+        DeleteBallot,
+        Login,
+        Logout,
+        Register
+    }
+
+    public class Event
+    {
+        public Event(EventType eventType, Guid pollId)
+        {
+            Timestamp = DateTime.Now;
+            EventType = eventType.ToString();
+            PollId = pollId;
+        }
+
+        public long Id { get; set; }
+
+        [Required]
+        public DateTime Timestamp { get; set; }
+
+        [Required]
+        public string EventType { get; set; }
+
+        [Required]
+        public Guid PollId { get; set; }
+
+        public string Value { get; set; }
```

Comment:
This seems to be used to show Http response codes primarily. Perhaps we should have another field for http response codes and leave them empty where not applicable. This should hopefully let us build up useful queries more easily. For example; "show me all events that have a non null httpcode" when we want to filter out page requests and the such.